site/content/contribute/server/feature-flags.md

```diff
@@ -35,7 +35,15 @@ Deployments to the managment system are overseen by the Cloud team. If you have
 
 ## Timelines for rollouts
 
-< Guidelines coming soon > 
+The feature flag is initially “off” and will be rolled out slowly. Note: this is an initial guideline and can be further iterated on over time.
+
+ - 1st week after feature is merged (T-30): 10% rollout to test servers; no rollout to customers.
```

Comment:
Little confused here: is this rolling out to 10% of test servers? Or rollout is at 10% and that means its on all test servers?

Reply:
Fixed the wording - Rollout is at 10% and that means it's on all test servers.